homeassistant/components/yeelight/light.py

```diff
@@ -413,8 +408,8 @@ async def _async_set_auto_delay_off_scene(entity, service_call):
 class YeelightGenericLight(YeelightEntity, LightEntity):
     """Representation of a Yeelight generic light."""
 
-    _attr_color_mode: str | None = COLOR_MODE_BRIGHTNESS
-    _attr_supported_color_modes: set[str] | None = {COLOR_MODE_BRIGHTNESS}
+    _attr_color_mode: ColorMode | str | None = ColorMode.BRIGHTNESS
```

Comment:
Can we drop the None typing here and the next line?

Reply:
Should be solved with new inheritance.